app/Connection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Connection extends Model
+{
+    //Return a list of connections available to you
+    public static function getAvailableConnections($user)
+    {
+        return self::all()->where('user_id', $user->id);
```

Comment:
This function, in every case, will read every record from the database.

Is this static function on the `Connection` object the best place to get all the connections for a `User`?

Reply:
fixed